android-components/components/ui/icons/src/main/res/drawable/mozac_ic_private_mode_12.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Spoke offline about adding this icon even though it's not part of the acorns-library

>I am wondering if we could avoid adding any icons that aren’t part of the [icon library](https://github.com/FirefoxUX/acorn-icons). It just makes our icons easier to be kept up to date with the latest UX designs if everything is in alignment with what UX is using. Worst case scenario is this one-off mask icon changes and we aren’t sure how to align it with the latest icon design. Is there’s a way we could use the 20x20 private mode icon [figma](https://www.figma.com/file/Kv7wWr8vH6AQrGhu9vmjdE/Mobile-Icons?type=design&node-id=2129-1788&mode=design&t=oUQvnSWYgaErCgJn-0) and [xml](https://github.com/FirefoxUX/acorn-icons/blob/main/icons/mobile/20/xml/ic_private_mode_circle_fill_20.xml) and add the round white border behind in the <layer>? We would also have to tint the icon to be the purple

In a followup, we will look into either:
(1) Using the 20x20 icon that is part of the icon library with a round white border in the `<layer>`
(2) Alternatively, get an icon from UX that will present us with the border.

Reply:
@gabrielluong filed https://bugzilla.mozilla.org/show_bug.cgi?id=1850641 to follow up. I'll NI you for the new icon